spec/features/users/viewing_users_spec.rb

```diff
@@ -5,8 +5,10 @@
 feature "View users" do
   context "as a system admin" do
     let(:user) { create(:user, system_admin: true) }
+    let(:dttp_user) { create(:dttp_user, provider_dttp_id: user.provider.dttp_id) }
```

Comment:
This could probably go into its own method below `and_there_is_a_dttp_user` to keep consistent with the spec

Reply:
All done